src/modality-specific-files/motion.md

```diff
@@ -0,0 +1,226 @@
+
+The extension is not limited to motion data in physical space but also encompasses simulated movement in virtual space, as far as these are comparable to movements in physical space. The extension is also not limited to the positions/orientations of human body parts, other dynamic objects (physical or virtual) in the environment
+whose motion is tracked may be included as additional indications of motion.
+
+For the current BEP, motion data coming from one tracking system MUST be stored in a `*_motion.tsv` file. A tracking system is defined as a group of tracking devices that share hardware properties (same recording device) and/or software properties (for example same sampling rates). For example if the position of multiple optical markers is processed via one recording unit, this would be defined as a tracking system. Note that it is not uncommon to have multiple tracking systems to record at the same time. Each tracking system should have its own `*tracksys-<label>_motion.tsv` file. One column in the `*tracksys-<label>_motion.tsv` file is intended to represent one data channel. The ordering of columns has to match the order of rows in the `*channels.tsv` file to allow a unambiguous assignment. All relevant metadata about a tracking systems is stored in accompanying sidecar `*tracksys-<label>_motion.json` file. The source data from each tracking system in their original format, if different from `.tsv`, can be stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data). The original data format might additionally hold more metadata than currently being specified in the `*_motion.json` file.
+
+When multiple tracking systems are used to record motion or motion capture is used alongside the collection of other BIDS modalities, is has to be possible to synchronise the recordings. A guideline to time synchronization between multiple modalities using recording onset and event time offset is described later in the specifications. To store the differences between recording onsets, `scans.tsv` files can be used.
+
+To store events which relate to a tracking system, it is recommended to use designated events files per tracking system. Such an events file name would include the tracksys key and look like `sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_tracksys-<label>_events.tsv`. The onsets in the event file relate can be related to the starting time of the tracking system in the `scans.tsv` file.
+
+### Sidecar JSON (`*_motion.json`)
+
+Generic fields (shared with other BIDS modalities) MUST be present:
```

Comment:
tables still need to be turned into marcos

Reply:
@JuliusWelzel did you keep the tables for now until you are sure that the macros work? or why did you leave them here?

Reply:
Yes, I could not get the Specs rendered with the schemas and left them in place for the moment